src/util/psbt/error.rs

```diff
@@ -37,8 +37,13 @@ pub enum Error {
     /// Magic bytes for a PSBT must be the ASCII for "psbt" serialized in most
     /// significant byte order.
     InvalidMagic,
+    ///Missing both the witness and non-witness utxo
+    MissingUtxo,
     /// The separator for a PSBT must be `0xff`.
     InvalidSeparator,
+    /// Triggered when output index goes out of bounds of when trying to get 
```

Comment:
This doesn't seem grammatically correct but as a non-native I'll defer this to @tcharding